### DISCUSSION TITLE
Setting fan curve based on GPU temperature sensor

### DISCUSSION THREAD

**Discussion by zurontos** [Rem0o/FanControl.Releases#1022] [Q&A]
So, I wanna go the route of attaching some fans plugged into my mobo to my GPU for temp testing (ignore my strange hobbies), and I'm wondering if there is any way to sync to GPU temp. I tried adding graphs and sensors, but all I can do is sync fan 2 and 3 to my current GPU fans, not it's temperature. I'm going this route because then I can just remove the fans easily from my motherboard and boot my system back on with a different set attached to compare separate results. Am I missing something or is this not yet a feature?

**Comment by Rem0o** [comment]
Just select your GPU core temp from your fan curve.
![image](https://user-images.githubusercontent.com/14118956/185632262-94d99f7b-05c0-4103-9f27-c170bdfc4d39.png)

**Reply by zurontos** [reply]
Can I make it use specific fans, like my system 1 and 2 fans? If so, how do I do that?? I tried to sync to temperature source and I couldn't figure how to do it
